e2e/tests/functional/plugins/lad/ladSet.e2e.spec.js

```diff
@@ -0,0 +1,68 @@
+/*****************************************************************************
+ * Open MCT, Copyright (c) 2014-2024, United States Government
+ * as represented by the Administrator of the National Aeronautics and Space
+ * Administration. All rights reserved.
+ *
+ * Open MCT is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+
+import { createDomainObjectWithDefaults } from '../../../../appActions.js';
+import { expect, test } from '../../../../pluginFixtures.js';
+
+test.describe('LAD Table Sets', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto('./', { waitUntil: 'domcontentloaded' });
+
+    const ladTableSet = await createDomainObjectWithDefaults(page, {
+      type: 'LAD Table Set'
+    });
+
+    const firstLadTable = await createDomainObjectWithDefaults(page, {
+      type: 'LAD Table',
+      parent: ladTableSet.uuid
+    });
+
+    await createDomainObjectWithDefaults(page, {
+      type: 'Sine Wave Generator',
+      parent: firstLadTable.uuid
+    });
+
+    const secondLadTable = await createDomainObjectWithDefaults(page, {
+      type: 'LAD Table',
+      parent: ladTableSet.uuid
+    });
+
+    await createDomainObjectWithDefaults(page, {
+      type: 'Sine Wave Generator',
+      parent: secondLadTable.uuid
+    });
+
+    await page.goto(ladTableSet.url);
```

Comment:
move this into the testcase

Reply:
@unlikelyzero @scottbell Note that `waitUntil: 'domcontentloaded'` is **only needed on initial page navigation**. If we're navigating via the router, we can use `page.goto(url)`, which is equivalent to `page.goto(url, { waitUntil: 'load' });`